f5_sphinx_theme/static/css/f5-theme.css

```diff
@@ -109,9 +109,9 @@ a.headerlink:hover {
   margin-right: 40px;
   margin-bottom: 256px;
   margin-top: 22px;
-  overflow-x: auto;
-  overflow-y: auto;
-  padding: 16px 12px 16px 16px;
+  overflow-x: scroll;
```

Comment:
Keep the padding settings, please.

Reply:
I'll email you a link to a build that shows what this is mean to fix.

Reply:
For record keeping purposes, what I did was remove the padding from #sidebar div and add it to .nav-sidebartoc so that the version_selector div would not have grey color padding around it. It still makes the nav menu look the same.